baybe/searchspace/continuous.py

```diff
@@ -391,14 +412,12 @@ def sample_uniform(self, batch_size: int = 1) -> pd.DataFrame:
 
         if not self.parameters:
             return pd.DataFrame(index=pd.RangeIndex(0, batch_size))
-
-        if (
-            len(self.constraints_lin_eq) == 0
-            and len(self.constraints_lin_ineq) == 0
-            and len(self.constraints_cardinality) == 0
-        ):
+        # If the space is completely unconstrained, we can sample from bounds.
+        if not self.is_constrained:
             return self._sample_from_bounds(batch_size, self.comp_rep_bounds.values)
 
+        # If there are neither cardinality nor interpoint constraints, we sample
```

Comment:
a bit confused by this comment, wehre is the check here done to make sure theres no interpoint?